specification.md

```diff
@@ -547,7 +595,20 @@ Returns this job's ID. The ID is assigned automatically by the
 implementation when the Job object is constructed. The ID is guaranteed
 to be unique on the client machine. The ID does not have to match the ID
 of the underlying LRM job, but is used to identify `Job` instances as
-seen by a client application.
+seen by a client application.  The ID can be used to later re-attach to the
+job with `Job j = executor.attach(native_job_id)`.
```

Comment:
This is inconsistent with the signature above. I would suggest simply linking to the relevant method; it will also make it more robust in the future should the signature change again.

Reply:
Thanks, fixed.